source/sass/mofo-bootstrap/overrides/_variables.scss

```diff
@@ -1,10 +1,11 @@
+// overrides for default Bootstrap variables
+// https://github.com/twbs/bootstrap/blob/v4.6.0/scss/_variables.scss
+
 $font-family-sans-serif: "Nunito Sans", Helvetica, Arial, sans-serif;
 $font-family-base: $font-family-sans-serif;
 
-$enable-flex: true; // Turn on Flexbox support
 $enable-transitions: true;
 
-$font-size-base: 1rem;
```

Comment:
Bootstrap's default value for this variable is `1rem` as well so we shouldn't have to override it.